### PR TITLE
 changes for priceBreakdown

### PR DESCRIPTION
**Pull request by dendoesit**
## Describe your changes

Added changes to priceBreakdown component to fit design

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

image

### CODE REVIEW

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 1]
```diff
@@ -13,35 +13,46 @@ export const CartPage = () => {
...
+      </Heading>
       <CartList products={products} />
       <Footer>
-        <PriceBreakdown currency="SEK" products={products} cost={cost} />
-        <Button fullWidth={true}>
-          <Link href={PageLink.cartReview()}>Check Out</Link>
-        </Button>
+        <Space y={1}>
+          <PriceBreakdown currency="SEK" products={products} cost={cost} />
+          <Button fullWidth={true}>
```
Comment (simonauner) [inline apps/store/src/components/CartPage/CartPage.tsx:25]: This should be sufficient
```suggestion
          <Button fullWidth>
```

File: apps/store/src/components/PriceBreakdown.tsx/PriceBreakdown.tsx [hunk 2]
```diff
@@ -98,4 +101,11 @@ const CollapsibleContent = styled(RadixCollapsible.Content)(() => ({}))
...
   borderTop: `1px solid ${theme.colors.gray300}`,
   height: theme.space[2],
+  marginTop: '0.75rem',
+  marginBottom: theme.space[3]
 }))
+
+const Price = styled.p(({ theme }) => ({
+  fontSize: '0.875rem',
+  color: theme.colors.gray600
+}))
```
Comment (simonauner) [inline apps/store/src/components/PriceBreakdown.tsx/PriceBreakdown.tsx:111]: Looks like you should add 

`"files.insertFinalNewline": true,`

to your VSCode config 📄

**Review by simonauner — COMMENTED** [review]
Nice, looks like the screenshot in the Jira ticket 🐱 
Just two minor remarks that doesn't change functionality or looks, just code cosmetics =)

File: apps/store/src/components/CartPage/CartPage.tsx [hunk 3]
```diff
@@ -13,35 +13,46 @@ export const CartPage = () => {
...
-  position: 'fixed',
+  position: 'absolute',
   bottom: 0,
   left: 0,
   width: '100%',
   padding: `0 ${theme.space[3]} ${theme.space[6]} ${theme.space[3]}`,
   a: {
     textDecoration: 'none',
   },
+  button: {
```
Comment (robinandeer) [inline apps/store/src/components/CartPage/CartPage.tsx:53]: Can we create a "styled button" instead => it's hard to know what this is targetting

File: apps/store/src/components/PriceBreakdown.tsx/PriceBreakdown.tsx [hunk 4]
```diff
@@ -98,4 +101,11 @@ const CollapsibleContent = styled(RadixCollapsible.Content)(() => ({}))
 const CollapsibleDivider = styled.div(({ theme }) => ({
   borderTop: `1px solid ${theme.colors.gray300}`,
   height: theme.space[2],
+  marginTop: '0.75rem',
+  marginBottom: theme.space[3]
 }))
+
+const Price = styled.p(({ theme }) => ({
+  fontSize: '0.875rem',
```
Comment (robinandeer) [inline apps/store/src/components/PriceBreakdown.tsx/PriceBreakdown.tsx:109]: nit: we now have fontSizes in the theme :)

**Review by robinandeer — APPROVED** [review]

**Pull request merged**